src/components/AppLayout/Sidebar/SafeHeader/index.tsx

```diff
@@ -145,8 +145,12 @@ const SafeHeader = ({
           <ButtonHelper onClick={onReceiveClick}>
             <Icon size="sm" type="qrCode" tooltip="Show QR" />
           </ButtonHelper>
-          <CopyToClipboardBtn textToCopy={address} />
-          <ExplorerButton explorerUrl={explorerUrl} />
+          <ButtonHelper>
+            <CopyToClipboardBtn textToCopy={address} />
```

Comment:
Why do we wrap a button with a button?

Reply:
This is not valid HTML

Reply:
A button cannot contain elements belonging to interactive category: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#interactive_content

Reply:
That's right. I'll fix it. Thanks